### PR TITLE
switch to docker command line in tests/e2e_node

### PR DESCRIPTION
**Pull request by dims**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
Avoid references to docker/docker code and use the command line instead.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

File: test/e2e_node/docker_util.go [hunk 1]
```diff
@@ -64,51 +54,41 @@ func isDockerLiveRestoreSupported() (bool, error) {
...
 
 // getDockerInfo returns the Info struct for the running Docker daemon.
-func getDockerInfo() (types.Info, error) {
-	var info types.Info
-	c, err := client.NewClient(defaultDockerEndpoint, "", nil, nil)
+func getDockerInfo(key string) (string, error) {
+	output, err := runCommand("docker", "info", "-f", "{{."+key+"}}")
 	if err != nil {
-		return info, fmt.Errorf("failed to create docker client: %v", err)
+		return "", fmt.Errorf("failed to get docker server version: %v", err)
```
Comment (oomichi) [inline test/e2e_node/docker_util.go]: Here should be `failed to get docker info` instead.
I guess the line came from line 31 ;-)

Comment (dims) [reply test/e2e_node/docker_util.go]: whoops! fixed.

**Review by neolit123 — COMMENTED** [review]
/lgtm

**Comment by dims** [comment]
@oomichi it's ready now.

**Comment by oomichi** [comment]
Thanks for updating,

/lgtm
/approve

**Pull request merged**